core/amber/src/main/scala/edu/uci/ics/texera/web/resource/ResultExportResource.scala

```diff
@@ -0,0 +1,48 @@
+package edu.uci.ics.texera.web.resource
+
+import com.typesafe.scalalogging.LazyLogging
+import edu.uci.ics.amber.core.virtualidentity.WorkflowIdentity
+import edu.uci.ics.texera.web.auth.SessionUser
+import edu.uci.ics.texera.web.model.websocket.request.ResultExportRequest
+import edu.uci.ics.texera.web.model.websocket.response.ResultExportResponse
+import edu.uci.ics.texera.web.service.WorkflowService
+import io.dropwizard.auth.Auth
+
+import javax.ws.rs._
+import javax.ws.rs.core.Response
+import scala.jdk.CollectionConverters._
+
+@Path("/export")
```

Comment:
Its better to name the root endpoint `result`, and rename the resource to be `ResultResource`